tools/alphafill_cofactor_survey/annotate_heteroatoms.py

```diff
@@ -52,8 +53,10 @@
     writer.writerows(annotated)
 
 with open(cofactor_only_output, "w") as f:
-    for ligand in cofactor_only:
-        f.write(f"{ligand}\n")
+    for lid in sorted(all_cofactors_from_json):
+        f.write(f"{lid}\n")
+  #  for ligand in cofactor_only:
+   #     f.write(f"{ligand}\n")
```

Comment:
can you remove these commented lines? If this is the previous version, we will still have it as commit on GitHub